CHANGELOG.md

```diff
@@ -9,7 +9,6 @@ Adding a new version? You'll need three changes:
 --->
 
  - [2.9.0](#290)
- - [2.8.1](#281)
```

Comment:
I guess we want to keep this?

Reply:
Yeah, rebase/merge cruft trying to make changes right after a release. Amended commit to fit with the current changelog to fix this and https://github.com/Kong/kubernetes-ingress-controller/pull/3318#discussion_r1073288085